### DISCUSSION TITLE
如何设置点击屏幕暂停

### DISCUSSION THREAD

**Discussion by fuxin-mail** [hooke007/mpv_PlayKit#479] [mpv 通用]
今天刚刚用mpv不知道怎么实现单击屏幕暂停

**Comment by hooke007** [comment]
https://github.com/hooke007/MPV_lazy/blob/9713164e5bd2342c55c15902c690fd8095393aa0/portable_config/input_uosc.conf#L109-L121

第109行就是鼠标左键
第121行就是暂停切换

**Comment by fuxin-mail** [comment]
如何让这个设置生效？把这行剪切到不映射横线上面吗？

**Reply by hooke007** [reply]
改
```
MBTN_LEFT         cycle pause
```

有这类需求建议看看快捷键怎么设置的
https://github.com/hooke007/MPV_lazy/blob/d06c7b8b1716cf6ad754fda14cf10ceba5faea1c/portable_config/input_easy.conf#L4

**Comment by fuxin-mail** [comment]
非常感谢！搞定。
